app_feup/lib/view/Pages/general_page_view.dart

```diff
@@ -104,6 +107,9 @@ abstract class GeneralPageViewState extends State<StatefulWidget> {
             },
             child: SvgPicture.asset(
               'assets/images/logo_dark.svg',
+              color: Theme.of(context).brightness == Brightness.light
+                  ? Theme.of(context).primaryColor
+                  : Colors.white,
```

Comment:
If you accept the dividing the icons colors, just change this to this
```suggestion
              color: Theme.of(context).accentIconTheme.color,
```